.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build (${{ matrix.ruby }} / ${{ matrix.os }})
+    strategy:
+      matrix:
+        ruby: [ 2.7, 2.6, 2.5, 2.4, truffleruby-20.3.0 ]
+        os: [ ubuntu-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: |
+        gem install bundler --no-document
+        bundle install
```

Comment:
Just a note regarding usage of ruby/setup-ruby in general:
I'd recommend using `bundler-cache: true` instead of this step, it would be faster and shorter.
(like done in https://github.com/ruby/mspec/blob/master/.github/workflows/ci.yml)